catalog/tests/dags/providers/provider_api_scripts/test_science_museum.py

```diff
@@ -6,24 +6,16 @@
 from catalog.tests.dags.providers.provider_api_scripts.resources.json_load import (
     make_resource_json_func,
 )
-from common.licenses import LicenseInfo
+from common.licenses import get_license_info
 from common.loader import provider_details as prov
 from common.storage.image import ImageStore
 from providers.provider_api_scripts.science_museum import ScienceMuseumDataIngester
 
 
-BY_NC_SA = LicenseInfo(
-    "by-nc-sa",
-    "4.0",
-    "https://creativecommons.org/licenses/by-nc-sa/4.0/",
-    None,
-)
-BY_SA = LicenseInfo(
-    license="by-sa",
-    version="4.0",
-    url="https://creativecommons.org/licenses/by-sa/4.0/",
-    raw_url=None,
-)
+BY_NC_SA_4_0 = get_license_info(license_="by-nc-sa", license_version="4.0")
+BY_NC_ND_4_0 = get_license_info(license_="by-nc-nd", license_version="4.0")
+BY_SA_4_0 = get_license_info(license_="by-sa", license_version="4.0")
```

Comment:
These `license_info` constants should probably be extracted.

Reply:
Agreed! There are a couple of other tests that define others that could be extracted as well.

Reply:
I opened a new issue: https://github.com/WordPress/openverse/issues/1978